_layouts/carousel-center.html

```diff
@@ -16,47 +16,47 @@ <h2>
-      {% endfor %}
-      </ol>
-      <div class="carousel-inner">
-        {% for card in page.carousel-cards %}
-          <div class="carousel-item {% if forloop.first %} active {% endif %}">
-            <div class="card mx-auto" style="width: 80vw; max-width: 70vh;">
-              {% if card.image %}
-                <div class="car-img-top" style="overflow: hidden;">
-                  <img src="{{ card.image }}" class="d-block card-img-top" alt="{{ card.alt }}" style="object-fit: cover; width: 80vw; height: 40vw; max-height: 40vh; max-width: 70vh; {% if card.imgstyle %}{{ card.imgstyle }}{% endif %}">
-                </div>
-              {% endif %}
-              <div class="card-body">
-                <p class="card-text">{{ card.quote }}
-                  {% if card.link %}
-                    <a href="{{ card.link }}">Mehr...</a>
-                  {% endif %}
-                </p>
-                {% if card.name %}
-                  <footer class="blockquote-footer mb-3">{{ card.name }}</footer>
-                {% endif %}
+        <div class="carousel-item {% if forloop.first %} active {% endif %}">
+          <div class="card mx-auto" style="width: 80vw; max-width: 70vh;">
+            {% if card.image %}
+              <div class="car-img-top" style="overflow: hidden;">
+                <img src="{{ card.image }}" class="d-block card-img-top" alt="{{ card.alt }}" style="object-fit: cover; width: 80vw; height: 40vw; max-height: 40vh; max-width: 70vh; {% if card.imgstyle %}{{ card.imgstyle }}{% endif %}">
               </div>
+            {% endif %}
+            <div class="card-body">
+              <p class="card-text">{{ card.quote }}
+                {% if card.link %}
+                  <a href="{{ card.link }}">Mehr...</a>
+                {% endif %}
+              </p>
+              {% if card.name %}
+              <footer class="blockquote-footer mb-3">{{ card.name }}</footer>
+              {% endif %}
             </div>
           </div>
-        {% endfor %}
-      </div>
-      <a class="carousel-control-prev mx-3" style="margin-top: 20vw; align-items: flex-start;" href="#{{ idname }}" role="button" data-slide="prev">
-        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
-        <span class="sr-only">Previous</span>
-      </a>
-      <a class="carousel-control-next mx-3" style="margin-top: 20vw; align-items: flex-start;" href="#{{ idname }}" role="button" data-slide="next">
-        <span class="carousel-control-next-icon" aria-hidden="true"></span>
-        <span class="sr-only">Next</span>
-      </a>
+        </div>
+      {% endfor %}
     </div>
+    <button class="carousel-control-prev"  href="#{{ idname }}" role="button" data-bs-slide="prev">
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </button>
+    <button class="carousel-control-next"  href="#{{ idname }}" role="button" data-bs-slide="next">
```

Comment:
```suggestion
    <button class="carousel-control-next"  href="#{{ idname }}" role="button" data-bs-slide="next" data-bs-target="#{{ idname }}>
```

Reply:
Buttons are not dark with the dark theme for some reason, unsure if this fixes it...

Reply:
Dark theme should mean light buttons, right?